css/styles.css

```diff
@@ -103,4 +124,18 @@ button {
     justify-content: flex-start;
     margin-left: 65px;
   }
+
+  [data-view="edit-profile"].hide {
```

Comment:
Instead of tying this to each specific situation, I would just make a class `.hide` with `display: none;` as its styling 👍